### PR TITLE
Stata appendix

### PR DESCRIPTION
**Pull request by kbjarkefur**
Stata appendix with Stata Style Guide

### CODE REVIEW

File: appendix/stata-guide.tex [hunk 1]
```diff
@@ -0,0 +1,271 @@
...
+\codeexample{stata-boilerplate.do}{./code/stata-boilerplate.do}
+
+\subsection{Before Saving}
+
+Similarly to boiler plate code there are good practices that should be followed before saving the data set.	
+These are sorting and ordering the data set, dropping intermediate variables that are not needed and 
+compressing the data set to save disk space and network bandwidth.
+
+If there is an ID variable or a set of ID variables, then the code should also test that they are uniqueally and 
+fully identifying the data set.\sidenote{https://dimewiki.worldbank.org/wiki/ID\_Variable\_Properties} ID
```
Comment (kbjarkefur) [inline appendix/stata-guide.tex]: @bbdaniels , I cannot get this URL to wrap in the margin, how did you do that?

**Comment by kbjarkefur** [comment]
@bbdaniels , apart from the question about a URL I asked in a review, I think this is done. I omitted a few things that I think are less important. Like, where to put the comma (although I like your rule and I followed it in my examples), but there is not enough time to go in to that level of detail in this version of the Style guide. I can do tomorrow if you think it is important!

File: appendix/stata-guide.tex (hunk 1 above)
Comment (bbdaniels) [reply appendix/stata-guide.tex]:
```suggestion
fully identifying the data set.\sidenote{\url{https://dimewiki.worldbank.org/wiki/ID\_Variable\_Properties}} ID 
```

Comment (bbdaniels) [reply appendix/stata-guide.tex]: Needs the `\url{}` tag -- added in suggestion

**Comment by bbdaniels** [comment]
Ok! I'll review.

**Comment by kbjarkefur** [comment]
Can this be merged now? I dont have the privileges to do so.

**Pull request merged**